python/lsst/rubintv/production/rubinTv.py

```diff
@@ -402,14 +401,14 @@ class MountTorqueChannel():
     """Class for running the mount torque channel on RubinTV.
     """
 
-    def __init__(self, location, doRaise=False):
+    def __init__(self, doRaise=False):
```

Comment:
I think I've previously complained that these could derive from a base class to handle all the duplicate code, so I'd like to be consistent and say the same thing again.

Reply:
Haha fair enough. You were probably right then, and you're probably right now, but this PR isn't about that 🙂